Client/src/Components/Dialog/index.jsx

```diff
@@ -0,0 +1,80 @@
+import LoadingButton from "@mui/lab/LoadingButton";
+import { Button, Modal, Stack, Typography } from "@mui/material";
+import PropTypes from "prop-types";
+const Dialog = ({ arialabelledby, ariadescribedby,
+    open, onClose, bdy1, yBtnLbl, yBtnOnClick, nBtnLbl, nBtnOnClick, theme, isLoading, bdy2 }) => {
+    return <Modal
+        aria-labelledby={arialabelledby}
+        aria-describedby={ariadescribedby}
```

Comment:
If you check out the docs for `aria-labelledby` and `aria-describevdy` you can see their purpose:

![image](https://github.com/user-attachments/assets/3c64f7d2-47d4-4244-9933-7055f916dc60)

so a good name for these props is probably `dialogLabelId` and `dialogBodyId` or something like that so that we know what these are for.

Reply:
In this case the MUI docs recommend using these: https://mui.com/material-ui/react-modal/#accessibility

I think the point is they implement the modal using non semantic markup, and need the wai-aria to fix accessibility for screen reades

Reply:
updated in : https://github.com/bluewave-labs/bluewave-uptime/pull/939/commits/21caca1ffbc9f624d9bc9a68283495f4278083af

Reply:
updated in : https://github.com/bluewave-labs/bluewave-uptime/pull/939/commits/21caca1ffbc9f624d9bc9a68283495f4278083af